src/sipnet/sipnet.c

```diff
@@ -375,7 +375,10 @@ typedef struct Parameters {
   double woodAllocation;  // fraction of NPP allocated to the roots
   double fineRootExudation;  // fraction of GPP exuded to the soil
   double coarseRootExudation;  // fraction of NPP exuded to the soil
-  // 6 parameters
+  // Calculated param
+  double coarseRootAllocation;  // fraction of NPP allocated to the coarse roots
```

Comment:
I think that `coarseRootFrac` is the parameter that needs to be added so that it can be initialized at emergence.